Tools-Override/Build.Common.targets

```diff
@@ -1,270 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I expect we may still need to make changes to this targets so I would keep it and only revert the one change.